### ISSUE TITLE
Allow time to be selected on mobile for WMS time layers

### ISSUE THREAD

**Issue by maltaesousa** [camptocamp/GeoMapFish#67]
### Who requested this new feature?
SITN

### Is your feature request related to a problem?
No

### Describe the solution you'd like
On desktop, times can be selected for wms time layers but not on mobiles. It would be nice to have both implementations: time picker and time slider for single or range times.

**Comment by remyguillaume** [comment]
@maltaesousa @pfanguin Do you mean something like this ? https://map.geo.bs.ch/s/K0G0
It's working at BS, bit I don't think de have made any big change for this.
If this is what you are looking for, I can look and share how we have configured it.

**Comment by pfanguin** [comment]
@maltaesousa ok perfect I understand. Thanks for you response !
@remyguillaume yes exactly !

**Comment by remyguillaume** [comment]
@pfanguin  @maltaesousa 

I just looked at how we solved this in BS, and actually we've just overridden some CSS : 

```
.gmf-layertree-node [ngeo-popover][ngeo-popover-dismiss] {
    display: flex;
}
```

But we are still in 2.4 for now. Perhaps this will be a little different in your version.

**Comment by maltaesousa** [comment]
Thanks @remyguillaume it works great! For our intance we needed to override CSS like this because the popover tilte was not being translated :

```css
.gmf-layertree-node [ngeo-popover][ngeo-popover-dismiss] {
  display: inline;
}

.popover-header {
  display: none;
}
```

@pfanguin tell me if this work around is ok for you. If it is, I will close this issue.

**Comment by rbovard** [comment]
@maltaesousa Is it possible to push it directly in GMF or it's too "hacky"?

**Comment by maltaesousa** [comment]
It's too hacky:
- Pressing on the clock icon is not easy: the icon is too small for fingers 
- Hiding popover-header because it's not translated is not a good solution
- Once you've selected the date, you have to press outside popover to close it. On mobile in portrait, you end up by clicking in layertree and activating another layer :/

**Comment by pfanguin** [comment]
@maltaesousa Yes it's work but we need some css to do this work around. It's could not be better to ask for a true integration ?

**Comment by maltaesousa** [comment]
Ok, so let's keep this open!
